File: Arrays/1281. Subtract the Product and Sum of Digits of an Integer/solution.swift
class Solution {
    func subtractProductAndSum(_ n: Int) -> Int {
        
        
        var product : Int = 1
        var totalSum : Int = 0
        
        for x in String(n){
            
            if let x = Int(String(x)) {
                product *= x
                totalSum += x
            }
        }
        
        
        return product - totalSum
    }
}



let n = 234
print(Solution().subtractProductAndSum(n))

File: Arrays/1281. Subtract the Product and Sum of Digits of an Integer/solution.py
class Solution:
    def subtractProductAndSum(self, n: int) -> int:
        

        product = 1
        total = 0

        for x in str(n):
            product *= int(x)
            total += int(x)
        
        return product-total


if __name__ == '__main__':

    """ Example 1: """
    n = 234
    print(Solution().subtractProductAndSum(n))